src/Templates/NewModule/build.psake.ps1

```diff
@@ -375,6 +410,7 @@ function PromptUserForCredentialAndStorePassword {
 }
 
 function AddSetting {
+    [System.Diagnostics.CodeAnalysis.SuppressMessage('PSShouldProcess', '', Scope='Function')]
```

Comment:
Nevermind, this file **is** under src. Duh.   Guess I need to just call it a night and get some sleep.  :-)